pyperformance/benchmarks/bm_tornado_http.py

```diff
@@ -86,6 +87,11 @@ def run_client():
 
 
 if __name__ == "__main__":
+    # https://github.com/tornadoweb/tornado/pull/2686
```

Comment:
Mention maybe pyperformance #61 issue: https://github.com/python/pyperformance/issues/61

The best would be to add a short sentence to explain this workaround ;-)